api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -449,3 +469,37 @@ message Plan {
   // successfully and data processing is complete.
   repeated TelemetryMetadata telmetry_metadata = 13;
 }
+
+// Request for the 'AddTle' method.
+message AddTleRequest {
+  // The ID of the satellite to add TLE for. The ID can be found on the StellarStation
+  // Console page for the satellite.
+  string satellite_id = 1;
+
+  // The first line of the TLE. Not a title line.
+  string tle_line_1 = 2;
```

Comment:
Let's go ahead and add an orbit package with a `Tle` message - wish we did it before adding such a message to `groundstation` but probably still a good idea to do this now.